Automattic/jetpack
# Prepare Boost for next dev cycle

Update readmes, versions, etc for next Boost dev cycle.

## Proposed changes:
* Update readmes, versions, etc for next Boost dev cycle.

### Other information:

## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:
* Look at the readme and changelog files, make sure they look ok.
* Make sure the versions look right for our next release after 1.8.0
